examples/vision/3d_multimodal_brain_tumor_segmentation.py

```diff
@@ -0,0 +1,707 @@
+            ignore_empty=True,
+            num_classes=num_classes,
+            name="dice",
+        ),
+        BinaryDiceMetric(
+            from_logits=True,
+            ignore_empty=True,
+            class_ids=[0],
+            num_classes=num_classes,
+            name="dice_tc",
+        ),
+        BinaryDiceMetric(
+            from_logits=True,
+            ignore_empty=True,
+            class_ids=[1],
+            num_classes=num_classes,
+            name="dice_wt",
+        ),
+        BinaryDiceMetric(
+            from_logits=True,
+            ignore_empty=True,
+            class_ids=[2],
+            num_classes=num_classes,
+            name="dice_et",
+        ),
+    ],
+)
+
+model.summary(line_length=100)
+
+"""
+## Callback
+
+We will be using sliding window inference callback to perform validation at certain interval or epoch during training. Based on the number of `epoch` size, we should set `interval` accordingly. For example, if `epoch` is set 15 and we want to evaluate model on validation set every 5 epoch, then we should set `interval` to 5.
+"""
+
+swi_callback_metric = BinaryDiceMetric(
+    from_logits=True,
+    ignore_empty=True,
+    num_classes=num_classes,
+    name="val_dice",
+)
+
+swi_callback = SlidingWindowInferenceCallback(
+    model,
+    dataset=val_ds,
+    metrics=swi_callback_metric,
+    num_classes=num_classes,
+    interval=5,
+    overlap=0.5,
+    roi_size=(96, 96, 96),
+    sw_batch_size=4,
+    save_path="brats.model.weights.h5",
+)
+
+"""
+## Training
+
+Set more epoch for better optimization.
+"""
+
+history = model.fit(train_ds, epochs=epochs, callbacks=[swi_callback])
+
+print(model.history.history.keys())
+his_csv = pd.DataFrame(model.history.history)
+his_csv.to_csv("brats.history.csv")
+his_csv.head()
+
+
+def plot_training_history(history_df):
+    metrics = history_df.columns
+    n_metrics = len(metrics)
+
+    n_rows = 2
+    n_cols = (n_metrics + 1) // 2  # ceiling division for columns
+
+    plt.figure(figsize=(5 * n_cols, 5 * n_rows))
+
+    for idx, metric in enumerate(metrics):
+        plt.subplot(n_rows, n_cols, idx + 1)
+        plt.plot(history_df[metric], label=metric, marker="o")
+        plt.title(metric)
+        plt.xlabel("Epoch")
+        plt.ylabel("Value")
+        plt.grid(True)
+        plt.legend()
+
+    plt.tight_layout()
+    plt.show()
+
+
+plot_training_history(his_csv)
+
+"""
+## Evaluation
+
+In this [Kaggle notebook](https://www.kaggle.com/code/ipythonx/3d-brats-segmentation-in-keras-multi-gpu/) (version 2), we trained the model on the entire dataset for approximately 25 epochs. The resulting weights will be used for further evaluation. Note that the validation set used in both the Colab and Kaggle notebooks is the same: `training_shard_36.tfrec`, which contains `8` samples. We will be evaluating the model per-class and on average.
+"""
+
+model_weight = gdown.download(id="1b-DpZkMsAX6I-niuLu7h8_y3XiaSHFyN", quiet=False)
```

Comment:
could you just upload these to kaggle? nice than gdown here (and we are already using kaggle)